<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="36335" started-at="2020-03-03T19:20:13 SGT" finished-at="2020-03-03T19:20:49 SGT">
    <groups>
    </groups>
    <test name="Test" duration-ms="36335" started-at="2020-03-03T19:20:13 SGT" finished-at="2020-03-03T19:20:49 SGT">
      <class name="testsuites.TestCase1">
        <test-method status="PASS" signature="initializeTestBaseSetup(java.lang.String, java.lang.String)[pri:0, instance:testsuites.TestCase1@42f93a98]" name="initializeTestBaseSetup" is-config="true" duration-ms="14619" started-at="2020-03-03T19:20:13 SGT" finished-at="2020-03-03T19:20:28 SGT">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[https://id.atlassian.com/login]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeTestBaseSetup -->
        <test-method status="PASS" signature="setUp(org.testng.ITestContext)[pri:0, instance:testsuites.TestCase1@42f93a98]" name="setUp" is-config="true" duration-ms="1" started-at="2020-03-03T19:20:28 SGT" finished-at="2020-03-03T19:20:28 SGT">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@a68df9]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testMethod1(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:testsuites.TestCase1@42f93a98]" name="testMethod1" duration-ms="21566" started-at="2020-03-03T19:20:28 SGT" finished-at="2020-03-03T19:20:49 SGT">
          <params>
            <param index="0">
              <value>
                <![CDATA[medel.vecina@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Testing1234]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Space2]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[did not expect to find [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at pagefactory.SpacePage.createSpace(SpacePage.java:139)
at testsuites.TestCase1.testMethod1(TestCase1.java:43)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMethod1 -->
      </class> <!-- testsuites.TestCase1 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
